src/pages/home/report/ReportActionCompose.js

```diff
@@ -742,6 +866,31 @@ class ReportActionCompose extends React.Component {
                     <ExceededCommentLength comment={this.comment} onExceededMaxCommentLength={this.setExceededMaxCommentLength} />
                 </View>
                 {this.state.isDraggingOver && <ReportDropUI />}
+                {!_.isEmpty(this.state.suggestedEmojis) && this.state.shouldShowSuggestionMenu && (
+                    <ArrowKeyFocusManager
+                        focusedIndex={this.state.highlightedEmojiIndex}
+                        maxIndex={getMaxArrowIndex(this.state.suggestedEmojis.length, this.state.isEmojiPickerLarge)}
+                        shouldExcludeTextAreaNodes={false}
+                        onFocusedIndexChanged={index => this.setState({highlightedEmojiIndex: index})}
+                    >
+                        <EmojiSuggestions
+                            isVisible={!_.isEmpty(this.state.suggestedEmojis)}
```

Comment:
Not sure why the `isVisible` prop is needed since whether or not the menu is rendered is controlled by the parent component.